packages/katello/katello/katello.spec

```diff
@@ -71,6 +70,8 @@ Requires: /usr/bin/psql
 Requires: /usr/bin/pg_dump
 Requires: /usr/bin/pg_dumpall
 
+Obsoletes: mongodb
```

Comment:
This feels like a strong statement to make. In the past we haven't obsoleted elasticsearch in this way.

Reply:
True, but we eliminated elasticsearch all together. In our case, we are wanting to force an upgrade. Further, the impression I was getting is that (https://github.com/Katello/katello-selinux/pull/15) will cause a conflict with the mongodb package?

Reply:
The last thing we want to have is both SCL Mongo and Mongo 2.6 on the system at once.

Reply:
If we add a Conflicts: in katello-selinux, would that remove the package as well or do we need to obsolete it somewhere?

Reply:
Conflicts will prevent installation and requires manual user intervention.